src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -915,6 +1258,41 @@ async def _handle_update_config(self, request_id: str, data: RTVIUpdateConfig):
         await self._update_config(RTVIConfig(config=data.config), data.interrupt)
         await self._handle_get_config(request_id)
 
+    async def _handle_update_context(self, data: RTVIAppendToContextData):
+        if data.run_immediately:
+            await self.interrupt_bot()
+        frame = LLMMessagesAppendFrame(messages=[{"role": data.role, "content": data.content}])
+        await self.push_frame(frame)
+        if data.run_immediately and self._context_aggregator:
```

Comment:
Hmm...isn't it theoretically possible that this `run_immediately` case is run before the `LLMMessagesAppendFrame` has reached a context aggregator and had the desired effect on the context?

Ah wait...I suppose it doesn't matter because when you then get a context frame using `get_context_frame()`, that frame has a reference to the "live" context rather than a snapshot of it at that point in time. So...we're probably good.

Reply:
Actually, the `LLMMessagesAppendFrame` might get processed after we do `self._context_aggregator.user().get_context_frame()`, so this might work sometimes and other it might not. I send a PR so we don't need to send a context frame, and therefore we don't need to pass the context aggregator to the constructor.